src/custom/components/AffiliateStatusCheck/index.tsx

```diff
@@ -87,19 +83,19 @@ export default function AffiliateStatusCheck() {
     isFirstTrade.current = true
   }, [referralAddress, chainId, account, fulfilledOrders.length, history, resetReferralAddress])
 
-  useEffect(() => {
-    async function handleReferralAddress(referralAddress: { value: string; isValid: boolean } | undefined) {
-      if (!referralAddress?.value) return
-      try {
-        const appDataHash = await uploadMetadataDocToIpfs(generateReferralMetadataDoc(referralAddress.value))
-        appDispatch(updateAppDataHash(appDataHash))
-      } catch (e) {
-        console.error(e)
-        setError('Affiliate program: There was an error while uploading your referral data. Please try again later.')
-      }
-    }
-    if (affiliateState === 'ACTIVE') handleReferralAddress(referralAddress)
-  }, [referralAddress, affiliateState, appDispatch])
+  // useEffect(() => {
+  //   async function handleReferralAddress(referralAddress: { value: string; isValid: boolean } | undefined) {
+  //     if (!referralAddress?.value) return
+  //     try {
+  //       const appDataHash = await uploadMetadataDocToIpfs(generateReferralMetadataDoc(referralAddress.value))
+  //       appDispatch(updateAppDataHash(appDataHash))
+  //     } catch (e) {
+  //       console.error(e)
+  //       setError('Affiliate program: There was an error while uploading your referral data. Please try again later.')
+  //     }
+  //   }
+  //   if (affiliateState === 'ACTIVE') handleReferralAddress(referralAddress)
+  // }, [referralAddress, affiliateState, appDispatch])
```

Comment:
I need to re-check I'm not missing anything from this flow now that the upload will be done during order placement rather than when following an affiliate link

Reply:
Yep, so you plan to delete the code once you are sure the affiliate cases are covered. 👍